contributing-tips.rst

```diff
@@ -0,0 +1,142 @@
+Tips for Making Good Contributions
+====================================
+
+.. _small-pr:
+
+Smallest meaningful PR
+------------------------
+
+A PR should normally address one issue. This makes it easier to review, easier to deploy, and easier to roll back in case of a problem. Additionally, the smaller the PR, the less likely it is to create a merge issue.
```

Comment:
This makes it easier to review, deploy and roll back in case of an **error**. -- error seems a better word while deployment and roll backs are considered.

Reply:
I disagree because "error" is a specific condition created by build process. We might roll back because something was factually wrong. Or because we want to organize something differently. Or lot so of other *problems*.